### PR TITLE
Remove selected nodes after they are deleted

### LINKED ISSUE #45: Selected Nodes stays in selectedNodeData permanently after being deleted
Hi,

When I try to update the selected nodes using a callback to modify either '`selectedNodeData`' or '`tapNode`' or '`tapNodeData`' is not reflecting on the front-end.  i.e. the selection highlight is not changing automatically.

Thanks,
Vivek

### PR DESCRIPTION
**Pull request by xhlulu**
## Pre-Merge checklist
- [x] The project was correctly built with `npm run build:all`.
- [x] If there was any conflict, it was solved correctly
- [x] All changes were documented in CHANGELOG.md.
- [x] All tests on CircleCI have passed.
- [x] All Percy visual changes have been approved.
- [x] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

## Reference Issues

Closes #45 

## Other comments

### CODE REVIEW

**Comment by shammamah** [comment]
@xhlulu For reference, if you put "Closes #45" in the body of the PR, it will automatically link the two and close the issue once this PR is merged :)

**Comment by xhlulu** [comment]
Done!

File: demos/usage-remove-selected-elements.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+])
+
+
+@app.callback(Output('cytoscape', 'elements'),
+              [Input('remove-button', 'n_clicks')],
+              [State('cytoscape', 'elements'),
+               State('cytoscape', 'selectedNodeData')])
+def remove_selected_nodes(_, elements, data):
+    if elements and data:
+        ids_to_remove = {ele_data['id'] for ele_data in data}
```
Comment (shammamah) [inline demos/usage-remove-selected-elements.py]: Should this be a list instead?

Comment (xhlulu) [reply demos/usage-remove-selected-elements.py]: A set has a O(1) retrieval time compared to lists :)

**Pull request merged**